projects/ui/src/lib/components/po-field/po-lookup/po-lookup-base.component.ts

```diff
@@ -243,6 +243,38 @@ export abstract class PoLookupBaseComponent
    */
   @Input('p-infinite-scroll') @InputBoolean() infiniteScroll: boolean = false;
 
+  /**
+   * @optional
+   *
+   * @description
+   *
+   * Ativa a funcionalidade de multipla seleção.
+   *
```

Comment:
Documentar como o model funcionará caso for multiplo, exemplo:
```

  Ativa a funcionalidade de múltipla seleção, com isso o valor do campo passará a ser uma lista de valores, por exemplo: `[ 12345, 67890 ]`
```  

Também alterar o filterService, informando a diferença caso o multiple estiver habilitado.